opennebula/resource_opennebula_virtual_network.go

```diff
@@ -198,15 +206,15 @@ func resourceOpennebulaVirtualNetwork() *schema.Resource {
 				Optional:      true,
 				Computed:      true,
 				Description:   "Carve a network reservation of this size from the reservation starting from `ip_hold`",
-				ConflictsWith: []string{"reservation_vnet", "reservation_size"},
+				ConflictsWith: []string{"reservation_vnet", "reservation_size", "reservation_ar_id", "reservation_first_ip"},
 				Deprecated:    "use 'hold_ips' instead",
 			},
 			"ip_hold": {
 				Type:          schema.TypeString,
 				Optional:      true,
 				Computed:      true,
 				Description:   "Start IP of the range to be held",
-				ConflictsWith: []string{"reservation_vnet", "reservation_size"},
+				ConflictsWith: []string{"reservation_vnet", "reservation_size", "reservation_ar_id", "reservation_first_ip"},
 				Deprecated:    "use 'hold_ips' instead",
 			},
 			"reservation_vnet": {
```

Comment:
this line should be added in `reservation_vnet` schema:
`Default: -1,`

Reply:
@treywelsh Just rebased this to the latest main branch. Can you give a little more detail on this comment?

Reply:
below in the code, there is this:

```
	reservation_vnet := d.Get("reservation_vnet").(int)

	// VNET reservation
	if reservation_vnet > -1 {
```
reservation_vnet will never be equal to `-1` if you don't set `-1` as a default value in the `reservation_vnet` schema

Reply:
ah! I see. Done :-)